src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './Introduce.scss';
+import { Link, useNavigate } from 'react-router-dom';
+
+const Introduce = () => {
+  return (
+    <div className="introduce">
+      <div className="wrapper">
+        <img className="profile-img" src="/images/cat.jpeg" alt="" />
+        <div className="profile-info">
+          <div className="name">김희연</div>
+          <div className="cohort-number">45기</div>
+          <Input />
+          <Button />
+          <div>
+            <Link to="/product">product 페이지로 이동</Link>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const handleInput = () => {
+  console.log('input 태그 동작');
+};
+
+export const Input = () => {
+  return <input type="text" className="input" onChange={handleInput} />;
+};
```

Comment:
```suggestion
export const Input = () => {
  const handleInput = () => {
  console.log('input 태그 동작');
};

  return <input type="text" className="input" onChange={handleInput} />;
};
```

컴포넌트를 분리하셨고, 해당 함수가 컴포넌트 내에서만 쓰이는 거라면
위와 같이 구현하시는 것이 좋습니다.